entity-api-spec.yaml

```diff
@@ -1,396 +1,1267 @@
+          description: The user is not authorized to retrieve the entity
+        '404':
           description: The target entity could not be found
-        "500":
+        '500':
           description: Internal error
-          
-  /collections:
+  '/collections/{id}':
     get:
-      summary: "Returns a list of Collections which include the basic Collection information and the uuids of all connected datasets.  If the component argument is omitted all Collections are returned.  For this call, the JWT is optional. If a valid token is provided with group membership in the HuBMAP-Read group any collection matching the id will be returned.  Otherwise if no token is provided or a valid token with no HuBMAP-Read group membership then only a public collection will be returned.  Public collections are defined as being published via a DOI (collection.doi_registered == true) and at least one of the connected datasets is public (dataset.metadata.data_access_level == 'public'). For public collections only connected datasets that are public are returned with it."
-      parameters:
-        - name: component
-          in: query
-          description: 'A prefix corresponding to the various components within the HuBMAP Consortium.'
-          required: false
-          schema:
-            type: string
-      responses:
-        "200":
-          description: successful operation
-        "401":
-          description: "The user's token has expired or the user did not supply a valid token"
-        "500":
-          description: Internal error
-  
-  /collections/{id}:
-    get:
-      summary: "Returns the information of the Collection specified by the uuid with all connected datasets.  For this call, the JWT is optional. If a valid token is provided with group membership in the HuBMAP-Read group any collection matching the id will be returned.  Otherwise if no token is provided or a valid token with no HuBMAP-Read group membership then only a public collection will be returned.  Public collections are defined as being published via a DOI (collection.doi_registered == true) and at least one of the connected datasets is public (dataset.metadata.data_access_level == 'public'). For public collections only connected datasets that are public are returned with it."
+      summary: 'Returns the information of the Collection specified by the uuid with all connected datasets.  For this call, the JWT is optional. If a valid token is provided with group membership in the HuBMAP-Read group any collection matching the id will be returned.  Otherwise if no token is provided or a valid token with no HuBMAP-Read group membership then only a public collection will be returned.  Public collections are defined as being published via a DOI (collection.doi_registered == true) and at least one of the connected datasets is public (dataset.metadata.data_access_level == ''public''). For public collections only connected datasets that are public are returned with it.'
```

Comment:
It says the `JWT`, are we using JWT token at all? Prob a typo.

Reply:
Yes, JWT doesn't belong there.  Looks like it was there before and I didn't remove it.